composer.lock

```diff
@@ -4398,7 +4398,8 @@
                 },
                 "patches_applied": {
                     "2774827 - node:group functionality for adding node:group:title": "https://www.drupal.org/files/issues/group_add-node-group-tokens-2774827-22.patch",
-                    "2984315 - Create an enabler that will allow Media to be added as group content, and access-controlled by group permissions.": "https://www.drupal.org/files/issues/2018-07-24/2984315-3.patch"
+                    "2984315 - Create an enabler that will allow Media to be added as group content, and access-controlled by group permissions.": "https://www.drupal.org/files/issues/2018-07-24/2984315-3.patch",
+                    "2842630 - Empty page when trying to create group node": "https://www.drupal.org/files/issues/2018-09-06/2842630-10.patch"
```

Comment:
This is kinda odd. The composer.lock file should not be picking up this change from this PR.